src/coverlet.core/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace Coverlet.Core.Extensions
+{
+    internal static class StringExtensions
```

Comment:
Instead of creating an Extension class instead put these methods in the `InstrumentationHelper` class. I made `DictionaryExtensions` because I was trying to mimic the experience already present in `netcoreapp` target framework

Reply:
I created that class to better adhere to the single responsibility principle. The `InstrumentationHelper` class already knows a lot...

Reply:
`Extension` methods should be used sparingly, only when you need to extend an object to mimic a certain behaviour. General utility methods are best used as plain methods. I'm fine with `InstrumentationHelper` being a sorta utility class for now, as long as each method performs a single function